src/pages/CvsPage/CvsPage.tsx

```diff
@@ -36,15 +61,21 @@ export const CvsPage = () => {
         <PageTopTypography title="CVs" caption="Cvs list" />
       </PageTop>
       <PageBody>
-        <Table
-          onDelete={handleItemDelete}
-          head={head}
-          items={cvs}
-          redirectButtonText="CV details"
-          deleteButtonText="Delete"
-          entryType={TableEntry.CV}
-          showNewEntryButton={true}
-        />
+        {loading
+          ? "loader"
+          : error
+          ? "error"
+          : data?.cvs && (
```

Comment:
Do not use nest ternary, try to create few simple constructions

Reply:
At first, I did it exactly the way you said. After a review, Rostik said I had to rewrite it with nested ternary. Perhaps, it is a matter of preference?

Reply:
Yea, it's not critical
I prefere simple constructions, Rostik prefere nested ternary - you can do it as you wish)